### ISSUE TITLE
error after installing: missing vbs directory and regList.wsf file

### ISSUE THREAD

**Issue by fernan** [mysql/mysql-shell-plugins#42]
I guess I successfully installed the 'MySQL Shell for VS Code 1.14.2+8.1.1 Preview', however there is an error showing up in the terminal output (see below). 

![image](https://github.com/mysql/mysql-shell-plugins/assets/175475/60c9fbef-839a-482e-aea5-d533c9885f5c)

```
C++ Runtime availability could not checked: Error: Command failed: cscript.exe //Nologo c:\Users\fernan\.vscode\extensions\oracle.mysql-shell-for-vs-code-1.14.2-win32-x64\out\vbs\regList.wsf A HKLM\SOFTWARE\WOW6432Node\Microsoft\VisualStudio\14.0\VC\Runtimes\X64 HKLM\SOFTWARE\Microsoft\VisualStudio\14.0\VC\Runtimes\X64
```

Upon further investigation it seems like there is no `vbs` directory under `out`, nor any file ending in `*.wsf`. Below is output from my WSL linux subsystem in Windows, and from File Explorer.

```
 [fernan] ls /mnt/c/Users/fernan/.vscode/extensions/oracle.mysql-shell-for-vs-code-1.14.2-win32-x64/out/
extension  frontend  main.cjs  main.js
[fernan] find /mnt/c/Users/fernan/.vscode/extensions/oracle.mysql-shell-for-vs-code-1.14.2-win32-x64/ -type f -name '*.wsf'
[fernan]
```

![image](https://github.com/mysql/mysql-shell-plugins/assets/175475/1a2ab216-4ecc-4482-9249-ef46fe3c4f65)

This is on: 

**Windows 10** Pro (Version 22H2; OS Build 19045.3803)

**Visual Studio Code:**
Version: 1.85.1 (user setup)
Commit: 0ee08df0cf4527e40edc9aa28f4b5bd38bbff2b2
Date: 2023-12-13T09:49:37.021Z
Electron: 25.9.7
ElectronBuildId: 25551756
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Windows_NT x64 10.0.19045

**Comment by jperna7254** [comment]
Seeing this same issue still, setup wizard cannot continue because there is no `vbs` directory and no `regList.wsf` file to execute to check that the VC++ runtimes are installed. This is completely preventing me from installing the extension on my Windows machine.

**Comment by Xo1otl** [comment]
I'm having the same problem on Windows. Can't use this extension because of the missing vbs directory and regList.wsf file. The setup wizard gets stuck trying to check for VC++ runtimes. This is completely blocking the installation on my Windows machine too.

**Comment by shafe123** [comment]
I ran into the same issue with some students of mine.  It seems like the latest version of the extension is not including the vbs utilities needed to check the registry keys.  You can fix this by grabbing the vbs folder from the [node-regedit](https://github.com/kessler/node-regedit) repository and putting it in the proper directory:  `extensions\oracle.mysql-shell-for-vs-code-1.14.2-win32-x64\out\vbs\`

**Comment by rennox** [comment]
Hi Guys

Thanks for your reports, we just published a new version which fixes the problem (1.16.4)

Please try with the latest version.
